src/java/org/apache/cassandra/tools/SSTableExport.java

```diff
@@ -119,6 +122,14 @@ public static CFMetaData metadataFromSSTable(Descriptor desc) throws IOException
         {
             builder.addClusteringColumn("clustering" + (i > 0 ? i : ""), header.getClusteringTypes().get(i));
         }
+
+        if (SecondaryIndexManager.isIndexColumnFamily(desc.cfname))
+        {
+            String index = SecondaryIndexManager.getIndexName(desc.cfname);
+            IndexMetadata indexMetadata =  IndexMetadata.fromSchemaMetadata(index, IndexMetadata.Kind.CUSTOM, null);
```

Comment:
Yes, you are right, first time I do not know what kind of the index should be ,so I just set it with COUSTOM as I think this do not make any sense for the code path. But I think I can now get the index meta info from system table and return the finally result.

Reply:
@adelapena .It seems hard for this tool to open a query for cql to select the index meta system table. So any advice on the kind of index ?

Reply:
I think that the kind of index doesn't have consequences for the sstable export, and I don't see a way to get the kind of index from the sstable. So it's probably ok to leave it as it is but, since it's not strictly correct, I'd leave a brief comment on the code about it.